apps/frontend/src/libs/modules/store/libs/middlewares/handle-unauthorized-error.middleware.ts

```diff
@@ -18,7 +18,11 @@ const handleUnauthorizedError = (): Middleware<
 				isRejected(action) &&
 				action.error.name === ExceptionName.UNAUTHORIZED
 			) {
-				void dispatch(authActions.logout());
+				if (action.type === authActions.signIn.rejected.type) {
```

Comment:
there should be a matcher for rejected actions
```suggestion
				if (isRejected(authActions.signIn)(action)) {
```